### DISCUSSION TITLE
Node.js debugging configuration

### DISCUSSION THREAD

**Discussion by zytact** [mfussenegger/nvim-dap#300] [Q&A] [answered]
Hello, first of all thanks for this awesome plugin.
I was configuring Node.js debugging by following [this](https://github.com/mfussenegger/nvim-dap/wiki/Debug-Adapter-installation#javascript). I got an error saying I must add ${workspaceFolder}/ prefix to program before ${file}. I dont't know if this specific to me but just wanted to point out. Thank you.

**Comment by mfussenegger** [comment, marked as answer]
Thanks for the pointer, I updated the example

(The wiki is btw. open for edit by anyone)

**Reply by zytact** [reply]
Oh, I had no idea, thank you :)
